src/Stryker.Core/Stryker.Core/Options/Inputs/OpenReporterInput.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using Stryker.Core.Exceptions;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public enum ReportType
+    {
+        None,
+        HTMLReport
```

Comment:
```suggestion
        None,
        Html
```

I don't think we need